docs/ecmwf-mars.md

```diff
@@ -30,12 +30,20 @@ To retrieve ECMWF MARS data set, navigate to the root directory
 cd ds-mapaction-ecmwf/
 ```
 
-and run
+and run one of the following options:
 
 ```bash
 poetry run python src/data_retrieval mars [country-iso]
 ```
 
+```bash
+poetry run python src/data_retrieval mars ETH --local /path/to/save
+```
+
+```bash
+poetry run python src/data_retrieval mars ETH --upload
+```
+
 for example to download the data for Ethiopia, run
```

Comment:
This explanation does not make sense. Please change to:

....

and run one of the following options:

to save file locally

```bash
poetry run python src/data_retrieval mars [country-iso] --local /path/to/save
```

or to upload to Azure

```bash
poetry run python src/data_retrieval mars [country-iso]  --upload
```

for example to download the data for Ethiopia and upload it to Azure, run

```bash
poetry run python src/data_retrieval mars ETH --upload
```